ZelBack/src/services/appsService.js

```diff
@@ -2759,10 +2753,9 @@ async function availableApps(req, res) {
       hddsuper: 5,
       hddbamf: 5,
       enviromentParameters: [`FOLD_USER=${userconfig.initial.zelid}`, 'FOLD_TEAM=262156', 'FOLD_ANON=false'],
-      commands: [
-      ],
+      commands: [],
       containerData: '/config',
-      hash: 'localSpecificationsVersion1', // hash of app message
+      hash: 'localSpecificationsFoldingVersion1', // hash of app message
```

Comment:
Don't we also have to adjust Folding@Home and Kadena hash, for the soft redeploy to happen with the new specs?

Reply:
I updated this hash because somewhere we are refferencing applications by hash. And I think we already used localSpecificationsVersion1 for KDA before. The hash should be strictly unique. It shall not cause any issue since that version of app does not exist on the network but just to be safe I changed it on this app.

Reply:
And yes. If we dont change the hash of kadena app and folding app, they will not be redeployed and will continue run according to old specifications

Reply:
KDA has a network update in 2 weeks so I was thinking we just leave it for now without changing the hash so nodes dont redeploy the app ( as there is ton of space on the network we do not really need it)